### DISCUSSION TITLE
Table of content - Title level 3

### DISCUSSION THREAD

**Discussion by borisdamevin** [vuejs/vitepress#880] [Q&A] [answered]
Hi,

It's possible to display level 3 title on table of content?

Thank you.

**Comment by xfq** [comment, marked as answer]
Do you mean something like https://github.com/vuejs/vitepress/pull/851 ?

**Reply by borisdamevin** [reply]
Yes, but on table of content not the sidebar.

**Reply by brc-dd** [reply]
@borisdamevin Set `outline: deep` in your frontmatter.

**Reply by Octobug** [reply]
For global setting:

```js
export default {
  themeConfig: {
    outline: 'deep',
  },
}
```
